graphdatascience/pipeline/pipeline_proc_runner.py

```diff
@@ -11,15 +15,39 @@
 class PipelineProcRunner(UncallableNamespace, IllegalAttrChecker):
     @client_only_endpoint("gds.pipeline")
     def get(self, pipeline_name: str) -> TrainingPipeline[PipelineModel]:
-        query = "CALL gds.beta.pipeline.list($pipeline_name)"
+        result = self.exists(pipeline_name)
+        if result["exists"]:
+            return self._resolve_pipeline(result["pipelineType"], result["pipelineName"])
+        else:
+            raise ValueError(f"No pipeline named '{pipeline_name}' exists")
+
+    def list(self, pipeline: Optional[TrainingPipeline[PipelineModel]] = None) -> DataFrame:
```

Comment:
As you maybe noticed, I tried to make these work backwards-compatibly; that is, a call to this endpoint on a server version < 2.5 would call the beta proc. But maybe that is overkill.